src/clr-core/badge/badge.element.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { registerElementSafely } from '@clr/core/common';
+import { html, LitElement } from 'lit-element';
+import { styles } from './badge.element.css';
+
+/**
+ * Web component badges.
+ *
+ * @noInheritDoc
+ * @element cwc-badge
+ * @slot default - Content slot for inside the badge
+ * @attr {color} - Sets the color of the badge from a predefined list of choices <br/> (`gray`, `purple`, `blue`, `orange`, `light-blue`)
+ * @attr {alias} - Sets the color of the badge from a predefined list of aliases <br/> (`1`, `2`, `3`, `4`, `5`)
+ * @attr {status} - Sets the color of the badge from a predefined list of statuses <br/> (`info`, `success`, `warning`, `danger`)
```

Comment:
The syntax here is not quite right, `@attr {String} color - Sets the color...` The value within `{ }` is the type of the attribute which is almost always String. This will regenerate the missing attr in the generated docs markdown files.

Reply:
Oh you are right. I'm not sure what happened there. Fixing.